docs/public/installation/prerequisites.md

```diff
@@ -36,14 +36,14 @@ at large cloud providers.
 
 Operating System | MariaDB | PostgreSQL
 ---------------- | --------| ---------------
-Debian 12        | 10.11   | 15.6
+Debian 12        | 10.11   | 15 ??
 Docker           | n/a     | n/a
-FreeBSD 14.1     | 8.0 (*) | 15.7
-Rocky Linux 8.10 | 10.3    | 10.23
-Rocky Linux 9.4  | 10.5    | 13.14
-Ubuntu 20.04     | 10.3    | 12.18
-Ubuntu 22.04     | 10.6    | 14.12
-Ubuntu 24.04     | 10.11   | 16.3
+FreeBSD 14       | 8.0 (*) | 16
+Rocky Linux 8    | 10.3??  | 10 ??
+Rocky Linux 9    | 10.5    | 13
+Ubuntu 20.04     | 10.3??  | 12 ??
```

Comment:
@tolvmannen. can you confirm the MariaDB and Postgresql versions for Ubuntu 20.04?